wallet/wallet.c

```diff
@@ -1460,7 +1457,7 @@ void wallet_channel_stats_incr_x(struct wallet *w,
 		break;
 	}
 
-	// TODO(vincenzopalazzo): Better here or in the default case of switch stmt?
+	// Sanity check!
```

Comment:
it'd be better to fold this change in with the commit that originally made the line.